package.json

```diff
@@ -148,6 +148,7 @@
     },
     "devDependencies": {
         "@changesets/cli": "^2.26.0",
+        "@prairielearn/prettier-plugin-sql": "workspace:^",
```

Comment:
I was actually waiting for your review on this to confirm that this is fine, since I noticed it's different than the other package reference. I didn't do anything fancy here, only `yarn add`, and it set the version as listed. I'll change it to `workspace:*`.

Reply:
Done in 8567fb516.